### ISSUE TITLE
Load and continue training NER model 

### ISSUE THREAD

**Issue by allhelllooz** [BrikerMan/Kashgari#228]
I am training a `BiGRU_CRF_Model` model with BERT with  1million data points and saving it.
```
### Load bert embeddings
bert_embed = BERTEmbedding(bert_location, task=kashgari.LABELING, sequence_length=bert_sequence_len)

### Define model
model = BiGRU_CRF_Model(bert_embed)
tf_board_callback = keras.callbacks.TensorBoard(log_dir='./logs', update_freq=1000)
eval_callback = EvalCallBack(kash_model=model, valid_x=valid_x, valid_y=valid_y, step=steps)

### Fit model
model.fit(train_x, train_y, valid_x, valid_y, batch_size=batch_size, callbacks=[eval_callback, tf_board_callback])
model.save("./BiGRU_CRF_BERT.model")
```

Then I am loading the model again to train it on some more data as shown below - 

```
from kashgari.utils import load_model
model = load_model('./BiGRU_CRF_BERT.model')
model.fit(train_x, train_y, valid_x, valid_y, batch_size=batch_size, callbacks=[eval_callback, tf_board_callback])
```

There is no issue in model load and testing.  But when I do `model.fit()` it gives me following error.

```
Traceback (most recent call last):
  File "BiLSTM_CRF_BERT.py", line 55, in <module>
    model.fit(train_x, train_y, valid_x, valid_y, epochs=epochs, batch_size=batch_size, callbacks=[eval_callback])
  File "/root/anaconda3/lib/python3.7/site-packages/kashgari/tasks/base_model.py", line 293, in fit
    **fit_kwargs)
  File "/root/anaconda3/lib/python3.7/site-packages/tensorflow/python/keras/engine/training.py", line 1433, in fit_generator
    steps_name='steps_per_epoch')
  File "/root/anaconda3/lib/python3.7/site-packages/tensorflow/python/keras/engine/training_generator.py", line 264, in model_iteration
    batch_outs = batch_function(*batch_data)
  File "/root/anaconda3/lib/python3.7/site-packages/tensorflow/python/keras/engine/training.py", line 1142, in train_on_batch
    self._assert_compile_was_called()
  File "/root/anaconda3/lib/python3.7/site-packages/tensorflow/python/keras/engine/training.py", line 2934, in _assert_compile_was_called
    raise RuntimeError('You must compile your model before '
RuntimeError: You must compile your model before training/testing. Use `model.compile(optimizer, loss)`.
```

How can I get this info about `optimizer` and `loss` from saved model ??

## Environment
- OS : Ubuntu 16.04
- Python Version: Conda 3.7.3

**Comment by allhelllooz** [comment]
Added code for model saving/training as well

**Comment by BrikerMan** [comment]
Just call `model.compile_model()` after loading the model.

```
from kashgari.utils import load_model
model = load_model('./BiGRU_CRF_BERT.model')
model.compile_model()
```

**Comment by allhelllooz** [comment]
Now I am getting following error

```
Traceback (most recent call last):
  File "BiLSTM_CRF_BERT.py", line 51, in <module>
    model.compile_model()
  File "/root/anaconda3/lib/python3.7/site-packages/kashgari/tasks/labeling/models.py", line 224, in compile_model
    kwargs['loss'] = self.layer_crf.loss
AttributeError: 'BiGRU_CRF_Model' object has no attribute 'layer_crf'
```

**Comment by BrikerMan** [comment]
> layer_crf

It seems we have ignored some issue related to the CRF layer. Will fix ASAP.

**Comment by allhelllooz** [comment]
Possibly! 
I tried few more things in between

To get `output_dim` so that I can pass it to CRF()
```
print("output_dim = ", len(model.pre_processor.label2idx))
```

Tried manually passing loss and metrics
```
layer_crf = CRF(output_dim=3, name='layer_crf')
model.compile_model(loss=layer_crf.loss, metrics=[layer_crf.viterbi_accuracy])
```
But compile step throws error.
Let us know about the updates on this issue. 
Thanks for reverting back fast.

**Comment by allhelllooz** [comment]
Should this issue be marked as **bug** ??

**Comment by BrikerMan** [comment]
Yep, this is a bug need to fix.

**Comment by allhelllooz** [comment]
@BrikerMan Is there any update. Let us know.

**Comment by BrikerMan** [comment]
This will fix in 0.5.4 version in several days. @allhelllooz
